solutions/search/semantic-search/using-openai-compatible-models.md

```diff
@@ -0,0 +1,185 @@
+---
+applies_to:
+  stack: ga
+  serverless: ga
+navigation_title: Using OpenAI compatible models
+---
+
+# Using OpenAI compatible models with the {{infer-cap}} API
+
+{{es}} enables you to use LLMs through the {{infer}} API, supporting providers such as Amazon Bedrock, Cohere, Google AI, HuggingFace, OpenAI, and more, as a service.
+It also allows you to use models deployed in your local environment that are compatible with OpenAI.
+
+This page walks you through the process of connecting local models to {{es}} using Ollama.
+
+[Ollama](https://ollama.com/) enables you to download and run LLM models on your own infrastructure.
+For a list of available models compatible with Ollama, refer to this [page](https://ollama.com/library).
+
+Using Llama models ensures that your interactions remain private, as the models run on your infrastructure.
```

Comment:
It is the platform that ensures privacy not the model (Llama is a family of models from meta).

```suggestion
Using Ollama ensures that your interactions remain private, as the models run on your infrastructure.
```